composer.json

```diff
@@ -10,7 +10,9 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~4.6",
-        "php-http/guzzle6-adapter": "^1.0"
+        "php-http/guzzle6-adapter": "^1.0",
+        "fabpot/php-cs-fixer": "^1.11",
+        "squizlabs/php_codesniffer": "^2.6"
```

Comment:
Do we need this? I suggest we just enable https://styleci.io/ and this will be solved for us.